app/src/main/java/com/lastaoutdoor/lasta/ui/screen/moreinfo/components/FriendSharePicker.kt

```diff
@@ -72,7 +72,16 @@ private fun FriendLazyColumn(
               Modifier.fillMaxWidth().padding(8.dp).testTag("shareTo" + friend.userId).clickable {
                 hideFriendPicker()
                 // Share the activity with the selected friend by sending its id
-                shareToFriend("|activity : |" + activityShared.activityId, friend.userId)
+                shareToFriend(
```

Comment:
maybe consider adding spaces between the "|" so that the text isn't glued together

"|" -> " | "

Reply:
I mean this is not really necessary because it is never shown to anyone, and is easier to split as a string and get as value